applications/nrf_desktop/src/util/Kconfig

```diff
@@ -23,6 +23,14 @@ config BT_ADV_PROV_SWIFT_PAIR
 config BT_ADV_PROV_FAST_PAIR
 	default y if BT_FAST_PAIR
 
+config BT_ADV_PROV_FAST_PAIR_SHOW_UI_PAIRING
+	depends on BT_ADV_PROV_FAST_PAIR
+	default n
```

Comment:
default n is not needed

Reply:
It's needed here, because we want to overlay the default of an existing Kconfig.